.editorconfig

```diff
@@ -0,0 +1,3 @@
+[*.sh]
```

Comment:
Did you mean to check this file in?

Reply:
Yeah, if you're gonna nitpick me on hard/soft tabs then I'd like the project to enforce the correct tabbing.